libqpdf/QPDFWriter.cc

```diff
@@ -1293,6 +1293,12 @@ QPDFWriter::willFilterStream(
         QTC::TC("qpdf", "QPDFWriter compressing uncompressed stream");
     }
 
+    bool empty_stream = stream_dict.getKey("/Length").getIntValue() == 0;
+    if (empty_stream) {
```

Comment:
It is preferable not to create the temporary variable `empty_stream` but to put the condition directly into the if statement `if (stream_dict.getKey("/Length").getIntValue() == 0) {...`. It makes no difference to the compiler, but to the human reader it raises the question whether something else in the method depends on it.

Reply:
Makes sense, fixed.